### DISCUSSION TITLE
Best Forum discussing *this* RP2040 core

### DISCUSSION THREAD

**Discussion by madias123** [earlephilhower/arduino-pico#1554] [Q&A]
Hello!
It seems like a silly question, so I have to backtrack:
Years ago I was involved into the STM32duino core (historical correct: libmaple) maintained by Roger Clark. At that time we decided to "break out" of the Arduino forum to found our own and to join forces: https://www.stm32duino.com/
At that time there was no "official" STM32 Arduino core and the forum was very manageable and informal.
STM32 is no longer interesting for me (too many variants and fake chips) and personally I just don't like the official version.
(now) In short:
Where is the main discussion group especially for this RP2040 core?
Arduino forum?
Raspberry forum?
reddit? (intended as a joke)
or here?

Thank you and best regards
Matthias

**Comment by earlephilhower** [comment]
FWIW, I myself really only work here on GH.

Occasionally I see links in from the Arduino.cc forums, but since Arduino published their own generic RP2040 (non Connect) MBED version I'm not sure it would be appropriate to use that.  The RPI forums are really focused only on the SDK (and maybe MicroPython since I believe RPI + the MicroPython guys have some sort of agreement).

There's a gitter room https://gitter.im/arduino-pico/community which is pretty light in traffic.  Sometimes my phone decides not to notify me about new messages from the app so I don't check there too often.
